src/mono/mono/mini/monovm.c

```diff
@@ -184,6 +184,9 @@ parse_properties (int propertyCount, const char **propertyKeys, const char **pro
 		} else if (prop_len == 27 && !strncmp (propertyKeys [i], "System.Globalization.UseNls", 27)) {
 			// TODO: Ideally we should propagate this through AppContext options
 			g_setenv ("DOTNET_SYSTEM_GLOBALIZATION_USENLS", propertyValues [i], TRUE);
+		} else if (prop_len == 32 && !strncmp (propertyKeys [i], "System.Globalization.AppLocalIcu", 32)) {
```

Comment:
can strlen be used here instead of using constant number?

Reply:
I was following what was done for other properties, I don't know if that should be kept separate from this change. /cc: @CoffeeFlux ?

Reply:
We're intentionally just calculating the length once to avoid doing it repeatedly in a sequence of string comparisons since the key is constant. I think there's minimal concern from the constant there; this looks fine to me.

Reply:
Thanks @CoffeeFlux.